dashboard/app/jobs/aichat_request_chat_completion_job.rb

```diff
@@ -41,7 +42,8 @@ class AichatRequestChatCompletionJob < ApplicationJob
     report_job_finish(request)
 
     # Raise an exception to notify our system of the failed job.
-    raise "AichatRequestChatCompletionJob failed with unexpected error: #{exception.message}. Context: #{request.to_json}"
+    message = exception.message && exception.message[0..MAX_ERROR_MESSAGE_LENGTH]
+    raise "AichatRequestChatCompletionJob failed with unexpected error: #{message}. Context: #{request.to_json}"
```

Comment:
@davidsbailey I think it's the length of the JSONified request we need to limit, not the exception.message. Do you think that's right based on the screenshots of the `last_error` entires from the `delayed_job` table below?

<img width="454" alt="Screenshot 2024-11-25 at 9 09 13 PM" src="https://github.com/user-attachments/assets/20292170-50af-43b5-96a5-a733b0b20283">
<img width="384" alt="Screenshot 2024-11-25 at 9 09 36 PM" src="https://github.com/user-attachments/assets/b973c00b-1ab8-4780-8d58-16ec7384bee3">

Reply:
Thanks Emily! Updated